.circleci/config.yml

```diff
@@ -0,0 +1,184 @@
+#
+#  Globality autogenerated CircleCI configuration
+#
+#  This file is auto generated with globality-build.
+#  You should not make any changes to this file manually
+#
+#  Any changes made to this file will be overwritten in the
+#  next version of the build.
+#
+#  See: http://github.com/globality-corp/globality-build
+#
+#
+
+defaults: &defaults
+  working_directory: ~/repo
+  docker:
+    - image: ${AWS_ECR_DOMAIN}/globality-build:fe815e7fe622e84fba3ea7c6f72299c4a052f414
+      aws_auth:
+        aws_access_key_id: ${AWS_ACCESS_KEY_ID}
+        aws_secret_access_key: ${AWS_SECRET_ACCESS_KEY}
+      environment:
+        EXTRA_INDEX_URL: "InjectedDuringRuntime"
+        AWS_ECR_DOMAIN: "InjectedDuringRuntime"
+        JFROG_AUTH: "InjectedDuringRuntime"
+
+deploy_defaults: &deploy_defaults
+  working_directory: ~/repo
+  docker:
+    - image: ${AWS_ECR_DOMAIN}/globality-build:fe815e7fe622e84fba3ea7c6f72299c4a052f414
+      aws_auth:
+        aws_access_key_id: ${AWS_ACCESS_KEY_ID}
+        aws_secret_access_key: ${AWS_SECRET_ACCESS_KEY}
+      environment:
+        EXTRA_INDEX_URL: "InjectedDuringRuntime"
+        AWS_ECR_DOMAIN: "InjectedDuringRuntime"
+        JFROG_AUTH: "InjectedDuringRuntime"
+
+whitelist: &whitelist
+  paths:
+    .
+
+version: 2
+
+jobs:
+  checkout:
+    <<: *defaults
+
+    steps:
+      - checkout
+
+      - run:
+          name: Install Python deps in a venv
+          command: |
+            python3 -m venv venv
+            . venv/bin/activate
+            pip install -U pip
+            pip install cookiecutter
+
+      - persist_to_workspace:
+          root: ~/repo
+          <<: *whitelist
+
+  build_base_docker:
+    <<: *defaults
+
+    steps:
+      - attach_workspace:
+          at: ~/repo
+
+      - setup_remote_docker
+
+      - run:
+          name: Build Base Docker
+          # install dependencies for loading ecs task definitions
+          command: |
+            . venv/bin/activate
+            ./scripts/instantiate.sh
+            (cd ml-papaya-extractor && globality-build build-gen local)
+            eval $(aws ecr get-login --no-include-email)
+            globality-build docker-base --repo ml-papaya-extractor --base-dir ml-papaya-extractor
+            globality-build docker --repo ml-papaya-extractor --base-dir ml-papaya-extractor
```

Comment:
These two commands leverage the `--base-dir` option added in globality-corp/globality-build#161.  This builds a docker image of the instantiated cookiecutter on which to run tests / lint / mypy